src/tray/vars/vault/main.yml.j2

```diff
@@ -1,7 +1,9 @@
 # customer: {{ customer }}
 # env_type: {{ env_type }}
 
-# ES database backend
+# ES data and lrs backends
+# RALPH_BACKENDS__LRS__ES__HOSTS: http://elasticsearch:9200
+# RALPH_BACKENDS__LRS__ES__INDEX: statements
```

Comment:
Shouldn't we delete the `RALPH_BACKENDS__DATA*` variables then?

Reply:
I'm not sure; however, it seems to me that the tray and helm chart uses both Ralph as an LRS and Ralph as a CLI (cronjob), thus they might need to define both variables

Reply:
Yes, you are totally right @SergioSim!